### PR TITLE
Fixed #23004 -- Hid sensitive environment parameters in debug view

### PR DESCRIPTION
**Pull request by MarkusH**

### CODE REVIEW

File: django/views/debug.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
     libraries={'i18n': 'django.templatetags.i18n'},
 )
 
-HIDDEN_SETTINGS = re.compile('API|TOKEN|KEY|SECRET|PASS|SIGNATURE', flags=re.IGNORECASE)
+HIDDEN_SETTINGS = re.compile('API|TOKEN|KEY|SECRET|PASS|SIGNATURE|URL|DSN', flags=re.IGNORECASE)
```
Comment (sergeykolosov) [inline django/views/debug.py]: I wonder, maybe in future we should introduce a `django.conf` setting for this regexp

Comment (apollo13) [reply django/views/debug.py]: Maybe, on the other hand forcing people to use a specific (relatively) small set of names for sensitive data might be actually not so bad.

Comment (sergeykolosov) [reply django/views/debug.py]: Also, https://github.com/django/django/blob/master/docs/ref/settings.txt#L1134 needs updating, it seems

Comment (sergeykolosov) [reply django/views/debug.py]: @apollo13 with that, I agree. Providing _some_ way to add more hidden items without too much of hack would be nice though.

**Comment by MarkusH** [comment]
@audiolion given #7996, would you be fine with me committing this PR if I credit you as a co-author? https://help.github.com/en/articles/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line . I'd like to bring this across the finish line.

**Comment by audiolion** [comment]
no problem man!

File: django/views/debug.py (hunk 1 above)
Comment (jdufresne) [inline django/views/debug.py]: `URL` seems a bit aggressive to me. I can imagine false positives will occur that would hide an interesting setting. For sensitive data, I agree with being better safe than sorry, but this one strikes me as too common. What do others think?

**Comment by carltongibson** [comment]
Closing as per ticket.

**Pull request closed without merging**